bigtable-hbase-parent/bigtable-hbase-integration-tests-2.x/src/test/java/com/google/cloud/bigtable/hbase/IntegrationTests.java

```diff
@@ -51,8 +51,11 @@
 })
 public class IntegrationTests {
 
+  public static final int TIME_OUT_MINUTES =
+      Integer.getInteger("integration.test.timeout.minutes", 10);
```

Comment:
Could you make it 30 minutes, at least for now?

Reply:
No.  If you need 30 minutes, please change it in the specific tests you care about.  Add `-Dintegration.test.timeout.minutes=30` to the command line.